docs/releases/4.0.txt

```diff
@@ -632,6 +632,11 @@ Moreover ``USE_L10N`` is deprecated as of this release. Starting with Django
 The :ttag:`{% localize %} <localize>` tag and the :tfilter:`localize`/
 :tfilter:`unlocalize` filters will still be honored by Django.
 
+Internationalization
+--------------------
+
+* Added support and translations for the Malay language.
```

Comment:
This is not a backward incompatible change but a new feature, so please move it below the [`Forms`](https://docs.djangoproject.com/en/4.0/releases/4.0/#forms):
```
Internationalization
~~~~~~~~~~~~~~~~~~~~

 * Added support and translations for the Malay language. 
```

Reply:
Moved `Internationalization` below `Forms` as requested.